frontend/src/pages/Play.jsx

```diff
@@ -28,11 +35,22 @@ export default function Play() {
     return () => pause();
   }, [isAnswerActive]);
 
+  const activateAnswer = (currentAnswer) => {
+    setIsAnswerActive(true);
+    if (currentAnswer) {
+      setScore(score + time);
+    }
+  };
+
   function nextLevel() {
     pickMovie(movieIdArray);
+    setCount(count + 1);
     reset();
     start();
     setIsAnswerActive(false);
+    if (count === 5) {
+      open();
+    }
```

Comment:
remember setCount is asynchronous: open will be called after `count`, not `count + 1`. is that what you want?